lib/galaxy/tool_util/linters/inputs.py

```diff
@@ -44,6 +44,12 @@ def lint_inputs(tool_xml, lint_ctx):
             if dynamic_options is None and len(select_options) == 0:
                 message = "No options defined for select [%s]" % param_name
                 lint_ctx.warn(message)
+
+            if param_attrib.get("display", "checkboxes") == "radio":
```

Comment:
I had another look at the code and the default is actually `None`:
```suggestion
            if param_attrib.get("display") == "radio":
```